src/include/sof/audio/codec_adapter/codec/cadence.h

```diff
@@ -36,6 +36,12 @@ struct cadence_api {
 	xa_codec_func_t *api;
 };
 
+enum default_stream_params_ids {
```

Comment:
If this is for a single or group of specific Cadence codec then we should say in the comments as users may try and use it for any Cadence codec. Can be incrementally fixed.

Reply:
@lgirdwood i need a clarification from @mrajwa. I think this default parameters IDS are actually for CODEC_WRAPPER which for Intel is the DAP algorithm.